### PR TITLE
Support for OCPP2.0.1 Plug&Charge

### PR DESCRIPTION
**Pull request by Pietfried**
## Describe your changes
Added support for OCPP2.0.1 Plug&Charge:
* Changed evse_security.yaml interface: Now two separate functions exist to get ocsp request data to be able to request the ocsp data for MO contract certificates
* LeafCertificateType enum was extended with the additional value: MO
* Refactored EvseSecurity and the lib/ocpp to address the required types/interface changes
* Added conversion function in OCPP201 module
* Changed EvseV2G module so that it always includes the OCSP request data of the contract certificate and optionally includes the certificate in the ProvidedIdToken
* Added P&C config for OCPP201

## Issue ticket number and link
requires:
* https://github.com/EVerest/libevse-security/pull/61
* https://github.com/EVerest/libocpp/compare/main...pg-c07-authorize-contract-certs (no PR yet)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [x] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

**Review by AssemblyJohn — APPROVED** [review]
The security part looks good to me.

File: modules/EvseV2G/iso_server.cpp [hunk 1]
```diff
@@ -1260,39 +1259,48 @@ static enum v2g_event handle_iso_payment_details(struct v2g_connection* conn) {
...
-                base64Buffer = static_cast<unsigned char*>(malloc(olen));
-                if ((base64Buffer == NULL) ||
-                    ((mbedtls_base64_encode(base64Buffer, olen, &olen, crt->raw.p, crt->raw.len)) != 0)) {
-                    dlog(DLOG_LEVEL_ERROR, "Unable to encode certificate chain");
-                    break;
-                }
-                contract_cert_chain_pem.append("-----BEGIN CERTIFICATE-----\n");
-                contract_cert_chain_pem.append(std::string(reinterpret_cast<char const*>(base64Buffer), olen));
-                contract_cert_chain_pem.append("\n-----END CERTIFICATE-----\n");
+        }
```
Comment (SebaLukas) [inline modules/EvseV2G/iso_server.cpp:1262]: Delete `else` because it's not needed

**Review by SebaLukas — APPROVED** [review]
The iso part looks good except one small thing (a not needed else)

**Comment by SebaLukas** [comment]
And clang-format has some problem

**Review by hikinggrass — APPROVED** [review]

**Pull request merged**